### ISSUE TITLE
ImportError: cannot import name 'app' from 'vllm.entrypoints.openai.api_server'

### ISSUE THREAD

**Issue by rdzotz** [JianyuZhan/vllm-on-sagemaker#1]
Thanks for putting this example together!

However It appears your example is running on an older version of vllm. ? 

I got this error when using the latest image and locally on vllm= 0.6.1

`from vllm.entrypoints.openai.api_server import app as api_app, parse_args, logger`

Could you let me know the version ?

**Comment by JianyuZhan** [comment]
yep, the 0.5.1, need to port to new version, the vllm  sever api  file (`vllm/entrypoints/openai/api_server.py`) has been overhauled since 0.5.2, a simple `run_server` method is incorporated, and I think this is better encapsulated for my project to use,  now a simple ` from vllm.entrypoints.openai.api_server import run_server` might be enough, I think.

**Comment by rdzotz** [comment]
Ok I will give it a go. Thanks for the pointers
